mfes/content/src/pages/content-details/[identifier].tsx

```
@@ -0,0 +1,179 @@
+                  variant="contained"
+                  sx={{
+                    bgcolor: '#49454F1F',
+                    color: '#1D1B20',
+                    margin: '3px',
+                    fontSize: '12px',
+                    backgroundColor: '#E9E9EA',
+                    borderRadius: '5px',
+                    boxShadow: 'none',
+                    textTransform: 'none',
+                  }}
+                >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)